cardano-db-sync/app/test-http-get-json-metadata.hs

```diff
@@ -112,6 +112,7 @@ classifyFetchError tf fe =
     OCFErrIOException {} -> tf {tfIOException = tfIOException tf + 1}
     OCFErrTimeout {} -> tf {tfTimeout = tfTimeout tf + 1}
     OCFErrConnectionFailure {} -> tf {tfConnectionFailure = tfConnectionFailure tf + 1}
+    _ -> tf
```

Comment:
hlint always complains about `_ ->` that's the only reason I've been doing `_otherwise` 🤷